src/modules/menu/components/share/MenuShareSection.vue

```diff
@@ -11,7 +11,13 @@
         <FontAwesomeIcon v-if="!shortLink" icon="spinner" spin size="2x" class="p-2" />
         <div v-if="shortLink" class="p-2">
             <MenuShareSocialNetworks :short-link="shortLink" class="pt-1" />
-            <MenuShareShortLinkInput :short-link="shortLink" class="px-2 py-3" />
+            <MenuShareInputCopyButton
+                :input-text="shortLink"
+                :label-text="$t('share_link')"
```

Comment:
would be better, as stated above, to use the same logic as with `copyText` and `copiedText`, and pass only the i18n key for this prop too, letting the component manage the translation itself when it is needed

Reply:
Done